run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+log() {
```

Comment:
May be worth considering keeping shell functions that are used in different scripts in a file that can be sourced by the scripts

Reply:
Sourcing now `/update-packages.sh` from `/run.sh` to get the log() function.